pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -126,12 +128,20 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
+	irEnabled, err := imageRegistryEnabled(adminRESTConfig)
```

Comment:
This determination is actually made by checking for endpoints I thik.  Check to see how the openshift-controller-manager-operator does it.  I don't think it's the capability.

Reply:
It does seem like we do use the capabilities, at least in this [internal image registry controller](https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/pkg/operator/internalimageregistry/management_state.go#L12) for openshift-controller-manager-oprator